### DISCUSSION TITLE
Blank page on first page with CSP error

### DISCUSSION THREAD

**Discussion by dev-nicelee** [outline/outline#1826] [Self Hosting] [answered]
I got an error below when I access to the first page..Is it something I have to setup more for this?🤔

```
Refused to load the script 'https://browser.sentry-cdn.com/5.22.3/bundle.min.js' because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-inline' 'unsafe-eval' gist.github.com". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

(I'm sorry if it has lack of information..)

**Comment by tommoor** [comment, marked as answer]
This is a small regression in v52 when you're not using Sentry to track errors, but it does not affect functionality or break anything other than printing that warning in the console. If the page is blank it's something else that's the cause.

**Reply by dev-nicelee** [reply]
@tommoor Yes, you're right. Internal server error I posted on [here](https://github.com/outline/outline/discussions/1824) was the actual problem. Thanks a lot tom! 👍
